api/system/v1alpha1/inter_cp_ping.proto

```diff
@@ -0,0 +1,16 @@
+syntax = "proto3";
+
+package kuma.system.v1alpha1;
+
+option go_package = "github.com/kumahq/kuma/api/system/v1alpha1";
+
+message PingRequest {
+  string instance_id = 1;
+  string address = 2;
+  uint32 inter_cp_port = 3;
+  bool ready = 4;
+}
+
+message PingResponse { bool leader = 1; }
+
+service InterCpPingService { rpc Ping(PingRequest) returns (PingResponse); }
```

Comment:
I was thinking the idea is to have an open bi-di stream between the instance and the leader, why did we move away from this idea?

Reply:
Why do we need a bi-directional stream?

Reply:
I'm very skeptical about this approach. Explicit request/response is much easier to debug and control.
Also about recent KDS fixes, I'd be cautious about using bidi streams...